libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts

```diff
@@ -26,6 +26,33 @@ The preset bottom margin has been removed from alert components. To implement th
         },
       ],
     },
+    {
+      name: '@skyux/layout',
+      components: [
+        {
+          name: 'description list',
+          styles: [
+            {
+              css: `
+.sky-description-list-vertical-mode .sky-description-list-content:last-child {
+  margin-bottom: 15px;
+}
+
+.sky-theme-modern .sky-description-list-vertical-mode .sky-description-list-content:last-child {
+  margin-bottom: 20px;
+}
+
+:root {
+  --compat-description-list-margin-bottom: 0;
```

Comment:
We should probably prefix this with `sky-` to keep it consistent with our CSS naming conventions.